triage-party/overlays/moc/config.yaml

```diff
@@ -595,6 +600,39 @@ rules:
     filters:
       - tag: similar
 
+  # Inconsistencies
+  active-unassigned:
+    name: "Lifecyle Active without assignee"
+    resolution: "Assign the person working on it or fix lifecycle status"
+    type: issue
+    filters:
+      - label: "lifecycle/active"
+      - tag: "!assigned"
+
+  long-active:
+    name: "Lifecycle Active but not updated for long"
+    resolution: "Update the status"
+    type: issue
+    filters:
+      - label: "lifecycle/active"
+      - updated: +14d
```

Comment:
shall we make this 10d, so that it is not spanning two sprints (at least formally not)?